packages/tooltip/src/components/tooltip/Tooltip.tsx

```diff
@@ -1,14 +1,84 @@
-import { Text } from "@stenajs-webui/core";
+import { Indent, Row, Text } from "@stenajs-webui/core";
+import {
+  Icon,
+  stenaExclamationCircle,
+  stenaExclamationTriangle,
+  stenaInfoCircle,
+} from "@stenajs-webui/elements";
+import TippyComponent, {
+  TippyProps as TippyComponentProps,
+} from "@tippyjs/react";
+import cx from "classnames";
 import * as React from "react";
-import { Popover, PopoverProps, PopoverVariant } from "../popover/Popover";
+import { TippyCallbackRef } from "../../hooks/UseTippyInstance";
+import styles from "./Tooltip.module.css";
 
-export interface TooltipProps extends Partial<Omit<PopoverProps, "content">> {
-  variant?: PopoverVariant;
+type TooltipVariant = "info" | "warning" | "error";
+
+interface TippyProps
+  extends Partial<Omit<TippyComponentProps, "theme" | "render">> {
+  tippyRef?: TippyCallbackRef<HTMLDivElement>;
+}
+
+export interface TooltipProps extends TippyProps {
+  variant?: TooltipVariant;
   label: string;
 }
 
-export const Tooltip: React.FC<TooltipProps> = ({ label, ...popoverProps }) => {
+const variantIcons = {
+  info: stenaInfoCircle,
+  warning: stenaExclamationCircle,
+  error: stenaExclamationTriangle,
+};
+
+export const Tooltip: React.FC<TooltipProps> = ({
+  visible,
+  trigger = "mouseenter",
+  children,
+  tippyRef,
+  delay = 0,
+  variant,
+  content,
+  maxWidth = "125px",
+  arrow = true,
+  label,
+  ...tippyProps
+}) => {
+  const textComponent = (
+    <Indent display={"inline-block"} indent={1}>
+      <Text color="white" size={"small"} variant="bold">
+        {label}
+      </Text>
+    </Indent>
+  );
+
   return (
-    <Popover {...popoverProps} content={<Text size={"small"}>{label}</Text>} />
+    <TippyComponent
+      interactive
+      className={styles.noPadding}
+      trigger={visible !== undefined ? undefined : trigger}
+      visible={visible}
+      theme={"dark"}
+      delay={delay}
+      maxWidth={maxWidth}
+      arrow={arrow}
+      content={
+        <Row spacing={0.5} indent={0.5} alignItems="center">
+          {variant ? (
```

Comment:
I think you might want to check for variant first and then do one Row or another, because the spacing will differ more than this. Right now, you only get 12px on the sides, but the wireframes say it should be 16px. If there is no variant, you could use `indent=2` and be done with it, but if there is a variant you need a bit more work. Just seems easier to me to not have to think about both scenarios at once :)

Also wondering how the variants would work in multirow. With `alignItems=center`, this is what you would get. Is that what we want?
<img width="138" alt="image" src="https://user-images.githubusercontent.com/6987939/207049110-fde279ab-d8b7-4e82-918b-eff72fdb351d.png">

Reply:
Checked with the design team and we would like this behaviour in case of multiple rows:
![image](https://user-images.githubusercontent.com/6987939/207310554-0cef905d-abce-4daf-a661-aaece912a369.png)

Reply:
The incorrect padding is still an issue.

Also, there is an issue with the icon alignment for one row atm:
![image](https://user-images.githubusercontent.com/6987939/208690691-0c842cd9-99d7-4432-a81c-dc46bc97613e.png)